CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,149 @@
+      1. Virtue of race or ethnicity; country of origin, or country of residence; gender(s), biological sex or sexual orientation; age or experience; economic standing or education; disability - physical or otherwise. Not listed are not excluded.
+   2. Harassment - online or otherwise are not tolerated. This includes any offensive, belittling or threatening behaviour toward an individual(s) or group(s).
+      1. Cultural diversity and language barriers may play a critical role in differentiating whether a perpetrator intentionally meant to be aggressive in their behaviour. Patience is crucial in investigating these cases.
+
+</details>
+
+2. <details><summary>Contribution does not entitle me or anyone else to act in any way that is not conducive to making this a productive and enjoyable environment</summary><br />
+
+    1. Status within the organisation or technical skill does not allow someone to be exempt from the rules.
+       1. The community does not owe any individual, irregardless of the size of their contributions or financial investments.
+    2. Members with greater responsibility (eg: moderators) are held to much higher expectations than others. We believe in leading by example.
+       1. The status of a community moderator is a privilege, but also a responsibility.
+       2. Moderators may step down if they do not feel they are able to uphold these responsibilities.
+    3. We should be patient with new or less experienced members; our contributions do not entitle us to belittle others (which would be indicative of harassment/discrimination)
+
+</details>
+
+3. <details><summary>We welcome new users; whether they be contributors, testers, documentation editors, or someone interested in our work. We will help those who need a hand getting started (within reason) with the same patience we would expect someone to give us.</summary><br />
+
+    1. The cost toward the community for onboarding new members should not be expected to be unreasonable. We will provide the resources and guidance for members to learn, however we cannot be expected to provide a teaching service.
+       1. Everyone starts at a different point and we will be patient in providing resources
+       2. Simple answers such as "look at the manual" are not tolerable; these questions and their answers will likely be viewed again in the future by users through search engines and should therefore be treated appropriately.
+       3. However, we recognise that it is not reasonable for repeated contributions to be fixed without any effort from the contributor to remedy their mistakes.
+    2. First impressions matter; we are a good community who will welcome and share with those who are willing to learn.
+       1. Failure to adapt can lead to an unreasonable cost to the community to maintain membership.
+       2. This may create situations which may seem contradictory. I understand however that this is in recognition of Code of Conduct item 4.
```

Comment:
I think we should not use "I" here, given that the document speaks for the community.